docs/development/setting-up.md

```diff
@@ -17,14 +19,8 @@ Current Node.js Maintenance LTS:
 - Node.js 10.x
 - NPM 6.x
 
-Unsupported Node.js LTS that should still work:
-- Node.js 8.13.0 or greater
-- NPM 6.x
-
-**Runtime**
-
-- Node.js `>=`4.x
-- `>=`ES5 compatible web browsers, IE11+
+`swagger-client` dev setup *might* also work on older Node.js versions, but we don't 
```

Comment:
I would say *might* work in dev setup browser environment. It will not work with tests or non-browser node.js env, due to `formdata-node` dependency requirements.

I recommend removing this line entirely.

Reply:
Agree, removing the sentence.